Miniscript-Template-000-Timelocks-in-Templates.mediawiki

```diff
@@ -0,0 +1,83 @@
+<pre>
+  Miniscript Template: 0
+  Title: Proposed Timelock Usage
+  Created: 2023-09-11
+</pre>
+
+
+1. Name of Template
+
+Proposed Timelock Usage
+
+2. Goal to be achieved by template
```

Comment:
In e2167a422b8504dad625b538d3b3c39d55e39640:

I'd change this to `=Motivation=` to match the style of other sections.